### PR TITLE
CI: Add compose tests via GitHub workflow

### PR DESCRIPTION
**Pull request by kalikiana**
Testing out a simple GHA to test the `docker-compose.yml` for the *webui* using [isbang/compose-action](https://github.com/marketplace/actions/docker-compose-action).

Related to #3755

### CODE REVIEW

**Comment by okurz** [comment]
Good idea, I don't see it executed though. Do we still need to enable something?

**Comment by kalikiana** [comment]
> Good idea, I don't see it executed though. Do we still need to enable something?

Unsure. I'm getting notifications with build results so it is being executed... somewhere

And I'm getting [this error](https://github.com/kalikiana/openQA/runs/2014727215?check_suite_focus=true):

```
Run isbang/compose-action@v0.1
Building with native build. Learn about native build in Compose here: https://docs.docker.com/go/compose-native-build/
Creating network "webui_default" with the default driver
Pulling data (openqa_data:)...
The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.
Continue with the new image? [yN]
Error: The operation was canceled.
```

And I get the same running `sudo docker-compose up` locally. @ilausuch can you confirm that the yml in master is not working?

File: Makefile [hunk 1]
```diff
@@ -280,6 +280,10 @@ tidy-js: check-js-beautify
 tidy: tidy-js
 	tools/tidy
 
+.PHONY: test-containers-compose
+test-containers-compose:
```
Comment (okurz) [inline Makefile:284]: Hm, that would be the first target that is not included when calling "make test" without any other arguments. I suggest to include it there as well. See how we currently split targets explicitly for different circleCI jobs but still have it included when one calls "make test" locally

Comment (kalikiana) [reply Makefile:284]: I was a bit worried to get pushback if I add more to `test` :-D I'll add it then

**Comment by kalikiana** [comment]
I'm not sure if requesting review here makes sense - since we were also discussing with @ilausuch and @okurz to propose this together with fixes needed to make compose work. But I'll make it "ready" now, and we see from there

File: Makefile (hunk 1 above)
Comment (okurz) [reply Makefile:284]: Which would mean you would need my changes first

**Review by ilausuch — APPROVED** [review]
As far as is commented about the test-checkstyle I approve that

**Comment by kalikiana** [comment]
Note that the "errors" in checkstyle are because now CircleCI runs the test, rather than just GHA

**Comment by kalikiana** [comment]
Closing in favor of pulling the test into #3773

**Pull request closed without merging**